plugins/services/src/js/structs/Service.js

```diff
@@ -116,12 +116,24 @@ module.exports = class Service extends Item {
       gpus = 0,
       disk = 0
     } = this.getSpec().getResources();
+    let executorCpus = 0;
+    let executorMem = 0;
+    let executorGpus = 0;
+    let executorDisk = 0;
+
+    if (this.getSpec().get("executorResources")) {
+      const executor = this.getSpec().get("executorResources");
+      executorCpus = executor.cpus ? executor.cpus : 0;
+      executorMem = executor.mem ? executor.mem : 0;
+      executorGpus = executor.gpus ? executor.gpus : 0;
+      executorDisk = executor.disk ? executor.disk : 0;
+    }
 
     return {
-      cpus: cpus * instances,
-      mem: mem * instances,
-      gpus: gpus * instances,
-      disk: disk * instances
+      cpus: (cpus + executorCpus) * instances,
```

Comment:
Isn't the correct calculation `(cpus * instances) + executorCpus` ? I can't imagine why we'd want to multiply executorCpus by instances

Reply:
According to the comments on the JIRA ticket  it is "per container"

Reply:
We may need more clarification because instances ≠ containers --  There may be one executor per container but not per instance container. Or maybe its per instance in which case this would be  correct!!

Reply:
@brandonc , if you create a pod with two instances and open it, you will see that each instance has executor resources.